README.md

```diff
@@ -1 +1,37 @@
-# hw01
+# Homework One (hw01)  
+## By James Seddon  
+
+### Goal: Edit ReadMe to Provide Introduction to Class
+_(as well as experimenting with R Markdown)_  
+
+
+## Introduction  
+
+Hello, dear viewer! My name is James Seddon, and I'm a 4<sup>th</sup> year in the college studying Public Policy and Political Science. I warn you that, unlike my [favorite facebook group](https://www.facebook.com/groups/1349827481747511), graphic design is not my passion. This should explain why this markdown document is **bold**, but not in a good way.  
+
+Here are three things you should know about me:  
+
+1. My actual passion is scrolling through twitter at all hours of the day or night, because as we've learned recently, political news never sleeps.  
+2. As a result of number one, my sleep schedule is now awfully screwed up.  
+3. Despite a limited understanding of science and math, I am somehow still extremely interested in space! This has led me to become an astronomy and astrophysics minor here.  
+
+#### Look, I made a thing. I am very proud of it.  
+
```

Comment:
There are a ton of markdown table generators so you don't always have to do this by hand! ([Here's one](https://jakebathman.github.io/Markdown-Table-Generator/))